.github/workflows/ci.yml

```diff
@@ -169,7 +169,7 @@ jobs:
       - name: Run Cargo Test
         if: matrix.test != 'false'
         # TODO: run llvm-cov only for single build since other builds are not sent to codecov anyway
-        run: cargo llvm-cov --workspace ${{ matrix.features }} --lcov --target ${{ matrix.target }} --output-path lcov.info
```

Comment:
Why did we change this?

Reply:
we wanted to sort req urls/bodies but only for tests.